src/iosrv/srv.pb-c.c

```diff
@@ -0,0 +1,105 @@
+}
+size_t srv__notify_ready_req__get_packed_size
+                     (const Srv__NotifyReadyReq *message)
+{
+  assert(message->base.descriptor == &srv__notify_ready_req__descriptor);
+  return protobuf_c_message_get_packed_size ((const ProtobufCMessage*)(message));
+}
+size_t srv__notify_ready_req__pack
+                     (const Srv__NotifyReadyReq *message,
+                      uint8_t       *out)
+{
+  assert(message->base.descriptor == &srv__notify_ready_req__descriptor);
+  return protobuf_c_message_pack ((const ProtobufCMessage*)message, out);
+}
+size_t srv__notify_ready_req__pack_to_buffer
+                     (const Srv__NotifyReadyReq *message,
+                      ProtobufCBuffer *buffer)
+{
+  assert(message->base.descriptor == &srv__notify_ready_req__descriptor);
+  return protobuf_c_message_pack_to_buffer ((const ProtobufCMessage*)message, buffer);
+}
+Srv__NotifyReadyReq *
+       srv__notify_ready_req__unpack
+                     (ProtobufCAllocator  *allocator,
+                      size_t               len,
+                      const uint8_t       *data)
+{
+  return (Srv__NotifyReadyReq *)
+     protobuf_c_message_unpack (&srv__notify_ready_req__descriptor,
+                                allocator, len, data);
+}
+void   srv__notify_ready_req__free_unpacked
+                     (Srv__NotifyReadyReq *message,
+                      ProtobufCAllocator *allocator)
+{
+  if(!message)
+    return;
+  assert(message->base.descriptor == &srv__notify_ready_req__descriptor);
+  protobuf_c_message_free_unpacked ((ProtobufCMessage*)message, allocator);
+}
+static const ProtobufCFieldDescriptor srv__notify_ready_req__field_descriptors[2] =
+{
+  {
+    "uri",
+    1,
+    PROTOBUF_C_LABEL_NONE,
+    PROTOBUF_C_TYPE_STRING,
+    0,   /* quantifier_offset */
+    offsetof(Srv__NotifyReadyReq, uri),
+    NULL,
+    &protobuf_c_empty_string,
+    0,             /* flags */
+    0,NULL,NULL    /* reserved1,reserved2, etc */
+  },
+  {
+    "nctxs",
+    2,
+    PROTOBUF_C_LABEL_NONE,
+    PROTOBUF_C_TYPE_UINT32,
+    0,   /* quantifier_offset */
+    offsetof(Srv__NotifyReadyReq, nctxs),
+    NULL,
+    NULL,
+    0,             /* flags */
+    0,NULL,NULL    /* reserved1,reserved2, etc */
+  },
+};
+static const unsigned srv__notify_ready_req__field_indices_by_name[] = {
+  1,   /* field[1] = nctxs */
+  0,   /* field[0] = uri */
+};
+static const ProtobufCIntRange srv__notify_ready_req__number_ranges[1 + 1] =
+{
+  { 1, 0 },
+  { 0, 2 }
+};
+const ProtobufCMessageDescriptor srv__notify_ready_req__descriptor =
+{
+  PROTOBUF_C__MESSAGE_DESCRIPTOR_MAGIC,
+  "srv.NotifyReadyReq",
+  "NotifyReadyReq",
+  "Srv__NotifyReadyReq",
+  "srv",
+  sizeof(Srv__NotifyReadyReq),
+  2,
+  srv__notify_ready_req__field_descriptors,
+  srv__notify_ready_req__field_indices_by_name,
+  1,  srv__notify_ready_req__number_ranges,
+  (ProtobufCMessageInit) srv__notify_ready_req__init,
+  NULL,NULL,NULL    /* reserved[123] */
```

Comment:
(style)  space required after that ',' (ctx:VxV)

(style)  space required after that ',' (ctx:VxV)